catalogue/webapp/pages/works/[workId]/images.tsx

```diff
@@ -47,10 +46,6 @@ const ImagePage: FunctionComponent<Props> = ({
   const title = work.title || '';
   const iiifImageLocation = image.locations[0];
 
-  // We only send a langCode if it's unambiguous -- better to send no language
-  // than the wrong one.
-  const lang = (work.languages.length === 1 && work.languages[0]?.id) || '';
```

Comment:
Can you remind me why that isn't required anymore, is it done at the viewer level?

Reply:
I'm just getting lang from the work when it's required, rather than doing it here and passing it along.